spec/services/post_alerter_spec.rb

```diff
@@ -343,6 +344,47 @@ def create_post_with_alerts(args = {})
     end
   end
 
+  context '@here' do
+    let(:topic) { Fabricate(:topic) }
+    let(:post) { create_post_with_alerts(raw: "Hello @here how are you?", user: tl2_user, topic: topic) }
+
+    before do
+      Jobs.run_immediately!
+    end
+
+    it 'does not notify unrelated users' do
+      expect { post }.to change(evil_trout.notifications, :count).by(0)
+    end
+
+    it 'does not work if user here exists' do
+      Fabricate(:user, username: SiteSetting.here_mention)
```

Comment:
I think this should error out if a user with a username of `SiteSetting.here_mention` do not already exist.